cmd/notation/sign.go

```diff
@@ -98,14 +93,6 @@ func runSign(command *cobra.Command, cmdOpts *signOpts) error {
 	}
 
 	// write out
-	path := cmdOpts.output
-	if path == "" {
-		path = dir.Path.CachedSignature(digest.Digest(desc.Digest), digest.FromBytes(sig))
-	}
-	if err := osutil.WriteFile(path, sig); err != nil {
-		return err
-	}
```

Comment:
Removing this block of code also removes the capability of outputting the signature to a specified local file path.

Reply:
Looks like we will not support local signing for this milestone. Then we always do remote (content) signing. Could you also remove the `--push` flag?

Reply:
Also, we need to remove `--push-reference` since we are not able to push signatures to other repos for the current workflow.